sdk/template-dpg/Azure.ServiceTemplate.Template/content/ci.yml

```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
```

Comment:
Does this need to live in the template, or is there a source of truth for this file elsewhere?

Reply:
When the service is first released, and there is no ci.yml file under /sdk/<service> folder, we will auto-generate this file if -includeCI parameter set

Reply:
the files under content/ directory are for CI. if customer pass -includeCI parameter, we will update those files and copy them to sdk/ <serive> folder to enable CI